docs/grading_lists/list_grader.md

```diff
@@ -14,7 +14,7 @@ grader = ListGrader(
 )
 ```
 
-Each element of answers is set as an answer that is passed as the answers key into the subgrader. This should be set up as two input boxes that the student types in.
+Each element of answers is set as an answer that is passed as the answers key into the subgrader. This should be set up as two input boxes that the student types in. Note that the `answers` key is provided as a python list for a `ListGrader`, and is a required key.
```

Comment:
> Note that the `answers` key is provided as a python list for a `ListGrader`, and is a required key.

Re "required": I think what you're getting at here is that multiple input graders cannot get their answers from `answer` or `expect` attributes; author must pass them directly to the grader with keyword arguments.

Do you know if that right? Maybe a statement to that effect goes better in `edx.md`

(The `answers` key is not truly required, e.g., if ListGrader is used inside another ListGrader.)